README.md

```diff
@@ -76,6 +76,7 @@ This plugin intends to support linting of ES2015+ (ES6+) import/export syntax, a
 * Limit the maximum number of dependencies a module can have ([`max-dependencies`])
 * Forbid unassigned imports ([`no-unassigned-import`])
 * Forbid named default exports ([`no-named-default`])
+* Prefer single named export declaration ([`group-exports`])
```

Comment:
Changed to _Prefer named exports to be grouped together in a single export declaration_